Archivist.lua

```diff
@@ -334,31 +399,29 @@ end
 -- DANGEROUS FUNCTION
 -- Your data will be lost. All of it. No going back.
 -- Don't say I didn't warn you
-function Archivist:DeleteAll(storeType)
+function proto:DeleteAll(storeType)
 	if storeType then
-		self.sv[storeType] = {}
+		self.sv.stores[storeType] = {}
 		for id, store in pairs(self.activeStores[storeType]) do
 			self.activeStores[storeType][id] = nil
 			self.storeMap[store] = nil
 		end
 	else
-		for id in pairs(self.prototypes) do
-			self.sv[id] = {}
-			self.activeStores[id] = {}
-		end
+		self.sv = {}
```

Comment:
This looks odd (in the sense that it isn't symetrical).

Shouldn't thius be self.sv.stores = {}